kit/azure/activity-log/main.tf

```diff
@@ -4,20 +4,56 @@ resource "azurecaf_name" "cafrandom_rg" {
   prefixes      = ["law"]
   random_length = 3
 }
-
+#
 # configure our logging subscription
 data "azurerm_subscription" "current" {
   subscription_id = var.subscription_id
 }
 
-# Creates a RG for LAW
+module "policy_law" {
+  source              = "github.com/meshcloud/collie-hub//kit/azure/util/azure-policies?ref=ef06c8d43611dd3bf6eebdd7f472b95472f86b0b"
+  policy_path         = "./lib/"
+  management_group_id = var.scope
+  location            = var.location
+
+  template_file_variables = {
+    default_location          = "${var.location}"
+    current_scope_resource_id = "${var.scope}"
+    root_scope_id             = "${var.scope}"
+  }
+}
+
+# Set up permissions for deploy user
+resource "azurerm_role_definition" "cloudfoundation_tfdeploy" {
+  name  = "${var.resources_cloudfoundation}_log_workspace"
+  scope = data.azurerm_subscription.current.id
+  permissions {
+    actions = ["Microsoft.Resources/subscriptions/resourcegroups/write"]
```

Comment:
we should actually take out all the logging related permissions from the bootstrap module

Reply:
changed it to read

Reply:
@JohannesRudolph u mean that stuff https://github.com/meshcloud/collie-hub/blob/3e5e93c17c7096a8755a57cffbd33f18a2ffa0ee/kit/azure/bootstrap/resources.deploy-spn.tf#L42C7-L53C26